sources/core/Stride.Core/Native/NativeLibraryHelper.cs

```diff
@@ -65,19 +67,54 @@ public static void PreloadLibrary(string libraryName, Type owner)
                         throw new PlatformNotSupportedException();
                 }
 
+                switch (Platform.Type)
+                {
+                    case PlatformType.Windows:
+                        extension = ".dll";
+                        break;
+                    case PlatformType.Linux:
+                        extension = ".so";
+                        break;
+                    case PlatformType.macOS:
+                        extension = ".dylib";
+                        break;
+                    default:
+                        throw new PlatformNotSupportedException();
```

Comment:
won't this throw on mobile?

Reply:
This method starts with `#if STRIDE_PLATFORM_DESKTOP`, so I'm assuming mobile will not invoke this code. I don't think we load native libs on mobile (and use some platform specific APIs)